agimus_controller_ros/agimus_controller_ros/agimus_controller_parameters.yaml

```diff
@@ -5,9 +5,24 @@ agimus_controller_params:
       type: double
       default_value: 0.01
       description: "Step time of the OCP in seconds"
+    dt_factor_n_seq:
+      factors:
+        type: int_array
+        default_value: [1]
+        description: " Number of dts between two time steps, the factor."
+        validation:
+          not_empty<>: []
+          lower_element_bounds<>: [0]
+      dts:
+        type: int_array
+        default_value: [19]
+        description: "Number of time steps, the n."
+        validation:
+          not_empty<>: []
+          lower_element_bounds<>: [0]
```

Comment:
@MaximilienNaveau out of curiosity, is there an easy way to make this a list of pai (factor, n_steps) instead of two lists?

Reply:
Unfortunately it's not super trivial. You can create a dictionnary like this:

```yaml
node_name:
  ros__parameters:
     dt_n_factor: [0, 1, 2]
     0:
        factor: 1
        ns: 2
    1:
        factor: 1
        ns: 2
   2:
        factor: 1
        ns: 2
```
which is super inconvenient.
So I guess that the 2 lists is the best option so far.

Reply:
This wouldn't work ?
```yaml
node_name:
  ros__parameters:
     dt_factor_n_seq:
       - [ 1, 10 ]
       - [2, 5]
```

Reply:
I believe in ROS1 that works but not in ROS2...

Reply:
After going through https://docs.ros.org/en/humble/Concepts/Basic/About-Parameters.html, indeed, it seems this is not possible in ROS2.